infra/nightly-resources/index.js

```diff
@@ -165,10 +187,17 @@ async function loadBenchmarks(compareTo) {
             if (!enabledModes.has(runMethod)) {
                 return undefined;
             }
+            // prevRun may be undefined
             const prevBenchmark = previousRun ? previousRun[benchName] : undefined;
-            const prevRun = prevBenchmark ? prevBenchmark[runMethod] : undefined;
+            if (prevBenchmark === undefined) {
+                addWarning(`Couldn't find a previous benchmark for ${benchName}`);
+            }   
+            const prevRunForMethod = prevBenchmark ? prevBenchmark[runMethod] : undefined;
+            if (prevBenchmark && prevRunForMethod === undefined) {
```

Comment:
why `=== undefined` in the previous if and just `prevBenchmark` here? I'm cool with exploiting the fact that undefined/null is false-y, but imo it should be standard. Either be explicit about `=== undefined` or not, but not both

Reply:
good idea